scripts/index.js

```diff
@@ -0,0 +1,133 @@
+const state = {
+    currentTemp: 70,
+    sky: 'Sunny',
+    defaultCity: 'Seattle'
+};
+
+const setTemperatureColor = (temp) => {
+    const temperatureText = document.getElementById("temperatureText");
+    if (temp >= 100) {
+        temperatureText.style.color = "darkred";
+    } else if (temp >= 80 && temp < 100) {
+        temperatureText.style.color = "red";
+    } else if (temp >= 70 && temp < 80) {
+        temperatureText.style.color = "orange";
+    } else if (temp >= 60 && temp < 70) {
+        temperatureText.style.color = "yellow";
+    } else if (temp >= 50 && temp < 60) {
+        temperatureText.style.color = "green";
+    } else if (temp > 32 && temp < 50) {
+        temperatureText.style.color = "teal";
+    } else if (temp <= 32) {
+        temperatureText.style.color = "blue";
+    }
+};
+
+const setLandscapeImage = (temp) => {
+    const landscapeImage = document.getElementById("landscapeImage");
+    if (temp >= 100) {
+        landscapeImage.src='./images/scorched-landscape.jpg';
+    } else if (temp < 100 && temp >= 80) {
+        landscapeImage.src='./images/desert-landscape.jpg';
+    } else if (temp < 80 && temp >= 70) {
+        landscapeImage.src='./images/grassy-landscape.jpg';
+    } else if (temp < 70 && temp >= 55) {
+        landscapeImage.src='./images/flower-landscape.jpg';
+    } else if (temp < 55 && temp >= 40) {
+        landscapeImage.src='./images/fall-landscape.jpg';
+    } else {
+        landscapeImage.src='./images/snow-landscape.jpg';
+    }
+};
+
+const increaseTemperature = () => {
+    state.currentTemp += 1;
+    const temperatureText = document.getElementById("temperatureText");
+    temperatureText.textContent = `${state.currentTemp}`;
+    setTemperatureColor(state.currentTemp);
+    setLandscapeImage(state.currentTemp);
+};
+
+const decreaseTemperature = () => {
+    state.currentTemp -= 1;
+    const temperatureText = document.getElementById("temperatureText");
+    temperatureText.textContent = `${state.currentTemp}`;
+    setTemperatureColor(state.currentTemp);
+    setLandscapeImage(state.currentTemp);
+};
```

Comment:
I like the use of helper functions, but I do want to note that these could be merged.

```suggestion
const changeTemp = (increment) => {
    state.currentTemp += increment;
    const temperatureText = document.getElementById("temperatureText");
    temperatureText.textContent = `${state.currentTemp}`;
    setTemperatureColor(state.currentTemp);
    setLandscapeImage(state.currentTemp);
};
```

You can then use an anonymous function as an event handler which calls it.

```javascript
increaseTempButton.addEventListener("click", () => increaseTemperature(+1)  );
```